src/napari_imagej/java.py

```diff
@@ -107,8 +107,16 @@ def initialize(self):
         ij_settings["ij_dir_or_version_or_endpoint"] = settings[
             "imagej_directory_or_endpoint"
         ].get(str)
-        ij_settings["mode"] = settings["jvm_mode"].get(str)
-        ij_settings["add_legacy"] = settings["include_imagej_legacy"].get(bool)
+        if hasattr(imagej, "gateway") and imagej.gateway:
```

Comment:
We're using this condition in multiple places. Can we refactor this into a variable, describing what it is doing? Something like "determine whether imagej has already been initialized"

Reply:
Done in 773fb2e